deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ev
+
+# Free Travis version does not support SSH key pairs, thus using encrypted files for SSH deploy key
+# https://docs.travis-ci.com/user/encrypting-files/
+openssl aes-256-cbc -K $encrypted_989f4ea822a6_key -iv $encrypted_989f4ea822a6_iv -in ansible/files/travis/master_key.enc -out master_key -d
+chmod 600 /tmp/deploy_rsa
+eval "$(ssh-agent -s)"
+ssh-add /tmp/deploy_rsa
+
+# Setup all epoch nodes with Ansible
+cd ansible && ansible-playbook --limit=epoch setup.yml
```

Comment:
I see this implements what is documented in [README.md](https://github.com/aeternity/infrastructure/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R21)

But I see the other preceding instructions, documented in README.md, neither in the `.travis.yml` nor in this `deploy.sh`. Why?
Those instructions are:
* `pip install -r ansible/pip-requirements.txt` - lack of this instruction would mean depending on packages already installed on Travis hence potentially changing over time
* `source ~/my/secrets/openstack.rc` - I understand this is preparatory for the next one;
* `ansible-inventory -i inventory/openstack.yml --list` - this would be useful for logging in CI hosts being recognized.

Reply:
I see that the documented `~/my/secrets/openstack.rc` may include pointers to the environment so you may want not to put it in clear in the repo and rather store it as encrypted with Travis encrypted var? (at the moment it is not in the repo at all.)

For the same reason you may want not to insert `ansible-inventory -i inventory/openstack.yml --list` logging call?

Reply:
README is meant more for manually testing, I'll update it.

- requirements.txt is [auto-picked by travis](https://docs.travis-ci.com/user/languages/python/#pip)
- indeed ~/my/secrets/openstack.rc is for local testing and it contains the sensitive openstack configuration. This configuration is set in travis environment settings (marked as secure)
- this information is still dumped during ansible playbook execution, so no need for additional requests here